samples/fetch_content/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ message(STATUS "FetchContent from branch: ${RYML_BRANCH_NAME}")
 
 include(FetchContent)
 FetchContent_Declare(ryml
-    GIT_REPOSITORY https://github.com/biojppm/rapidyaml.git
+    GIT_REPOSITORY https://github.com/${PULL_REQUEST_HEAD_REPO:biojppm/rapidyaml}.git
```

Comment:
I'm not super familiar with Make's syntax, so feel free to edit. I want to use `PULL_REQUEST_HEAD_REPO` and fallback to `biojppm/rapidyaml` if `PULL_REQUEST_HEAD_REPO` is not set.

Reply:
This should get you on your way: https://cmake.org/cmake/help/latest/variable/ENV.html